src/utils/jsonwriter.cpp

```diff
@@ -64,6 +38,32 @@ std::string escape(const std::string_view in)
     return out;
 }
 
+void AsmParser::JsonWriter::writeKeyName(const char *key)
+{
+    this->out << "\"" << escape(key) << "\": ";
```

Comment:
The change is in this line and #48. The rest comes from moving the `escape(std::string_view)` function to the top of the file before the constructor